filesystem/src/test/java/org/frankframework/filesystem/LocalFileSystemActorTest.java

```diff
@@ -22,23 +23,33 @@
 public class LocalFileSystemActorTest extends FileSystemActorTest<Path, LocalFileSystem> {
 
 	@TempDir
-	public Path folder;
+	public Path temp;
+	private Path root;
+
+	@BeforeEach
+	@Override
+	public void setUp() throws Exception {
+		// Append a folder to the test-dir, to make sure that using a root folder
+		// that doesn't exist yet will not fail.
+		root = temp.resolve("testLocalFSRoot");
+		super.setUp();
+	}
```

Comment:
Should this dir also be deleted in the `@AfterAll`?

Reply:
De `@TempDir` annotatie verwijderd de gemaakte temporary directory met hele inhoud al na afloop test, dus nee dat hoeft niet.